docs/template-migration/phases/active-migration/ai-template-migration-guide.md

```diff
@@ -0,0 +1,492 @@
+## Overview
```

Comment:
Did you mean to commit this file in this PR?

Reply:
Nope! Good catch